packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -71,16 +72,17 @@ class BottomAppBar extends StatefulWidget {
   /// Defaults to 8, the appropriate elevation for bottom app bars.
   final double elevation;
 
-  /// Whether to make a notch in the bottom app bar's shape for the floating
-  /// action button.
+  /// The notch that is used to make a notch for the floating action
```

Comment:
this sentence doesn't really make sense any more. The first noun is probably wrong? Maybe "The delegate"?

Reply:
Delegate would make sense, but [hopefully] I simplified the entire sentence a bit.